src/DynamoCore/Configuration/Configurations.cs

```diff
@@ -251,9 +251,9 @@ public class Configurations
 
         #region Externally Visible Strings
 
-        internal static readonly string SessionTraceDataXmlTag = "SessionTraceData";
-        internal static readonly string NodeTraceDataXmlTag = "NodeTraceData";
-        internal static readonly string CallsiteTraceDataXmlTag = "CallsiteTraceData";
+        internal static readonly string SessionTraceDataTag = "Bindings";
```

Comment:
Good point. I think unfortunately, yes. `LoadTraceDataFromXmlDocument` will still be used for legacy dyn opening and it will be checking this tag from the old dyns. I will revert the related changes. For the new json block `bindings`, I can hardcode the json block name just like the other ones. But should we also consider making a config file to store all these json property names? @ikeough